packages/streams/n2k-signalk.js

```diff
@@ -108,6 +110,15 @@ ToSignalK.prototype.isFiltered = function (source) {
   )
 }
 
+ToSignalK.prototype.filterSelfAISStatic = function (values) {
+  return values.filter((kv) => {
+    return (
+      kv.path !== '' &&
```

Comment:
How is this supposed to work considering that n2k-signalk outputs deltas using the empty path trick? https://github.com/SignalK/n2k-signalk/blob/3b0401af0738c3fd8097ac661e060cdf4b9296cc/test/129794_ais_class_a_static_data.js#L59-L77

Reply:
It's currently just ignoring that data for self.

Reply:
The baseDeltas check does nothing if (did not check..) all the static data from AIS has `path: ''`. Or am I missing something?